lib/shared/config/cli-flags.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var yargs = require('yargs');
+var camelCase = require('lodash.camelcase');
+
+function mergeToCliFlags(cliFlags, config, cliOptions) {
+  var argv = yargs(process.argv.slice(2)).argv;
+  var cfgFlags = config.flags || {};
+
+  var optNames = ['silent'];
+  optNames.filter(excludeArgsByUser).map(camelCase).forEach(copyConfig);
```

Comment:
Because property names of `cliFlags` (=`opts` in `index.js`) are camel case, though property names of `argv` (command-line flag name) are hyphenated name.